be/src/exec/vectorized/aggregate/aggregate_blocking_node.cpp

```diff
@@ -14,22 +14,23 @@ Status AggregateBlockingNode::open(RuntimeState* state) {
     RETURN_IF_ERROR(exec_debug_action(TExecNodePhase::OPEN));
     SCOPED_TIMER(_runtime_profile->total_time_counter());
     RETURN_IF_ERROR(ExecNode::open(state));
-    RETURN_IF_ERROR(Expr::open(_group_by_expr_ctxs, state));
-    for (int i = 0; i < _agg_fn_ctxs.size(); ++i) {
-        RETURN_IF_ERROR(Expr::open(_agg_expr_ctxs[i], state));
+    RETURN_IF_ERROR(Expr::open(_aggregator->group_by_expr_ctxs(), state));
+    for (int i = 0; i < _aggregator->agg_fn_ctxs().size(); ++i) {
+        RETURN_IF_ERROR(Expr::open(_aggregator->agg_expr_ctxs()[i], state));
```

Comment:
```
    RETURN_IF_ERROR(Expr::open(_aggregator->group_by_expr_ctxs(), state));
    for (int i = 0; i < _aggregator->agg_fn_ctxs().size(); ++i) {
        RETURN_IF_ERROR(Expr::open(_aggregator->agg_expr_ctxs()[i], state));
    }
```
refactor: define a Aggregator::open function and put code above into open.

Reply:
I'll fix it next pr